moped-editor/src/utils/taskOrderComponentHelpers.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { TextField } from "@material-ui/core";
+
+/*
+ * Socrata Endpoint
+ */
+export const SOCRATA_ENDPOINT =
+  "https://data.austintexas.gov/resource/ggav-ufvc.json?$limit=9999";
```

Comment:
Is the 9999 value for limit intended to prevent enormous reset sets from coming back from Socrata or to ask that it give you every conceivable task order out the `[TEST] Mobility Project Task Orders` dataset. Given that there are currently 2,650 records total in the dataset, I presume it's the latter. 

If that's the case, I believe you can eliminate the limit argument. Socrata 2.1 endpoints have a default limit value of 'unlimited,' [according to this page](https://dev.socrata.com/docs/queries/) in their documentation. Socrata indicates that this dataset is serving up data using the latest version of their API. 

No matter the answers to the questions above, this still works and is 👍

Reply:
We have the same 9999 pattern in `signalComponentHelpers.js`, when we query that socrata endpoint, just fyi

Reply:
Interesting what you cited about Socrata's latest API documentation Frank. I added this limit because when I was working on this it was limiting me at the first 1,000. This is what I think you'll see if you [visit this URL](https://data.austintexas.gov/resource/ggav-ufvc.json) 

![Screen Shot 2022-01-04 at 3 49 00 PM](https://user-images.githubusercontent.com/5697474/148128502-f496caae-12e5-4744-889a-5a09942149db.png)

This auto-enforced limit of 1000 has been a common experience in my work with the SODA API but I would love if they changed it to `unlimited`, but I guess seeing is believing for me.

Reply:
Good catch on the docs @frankhereford - I take that to mean there is no upper limit on how many records you can request, but you must specify a limit if you want more than the default (1k). That squares w/ my experience querying Socrata.

Reply:
@johnclary @mateoclarke - You're both so right! I misinterpreted the parenthetical in the "Default" column in those docs to mean that on 2.1 endpoints, the default would be unlimited, but as you pointed out, it is actually a default of 1K with the _option_ to turn it up to whatever amount of data you want returned. Thanks for clarifying that for me - TIL! 🙏